### DISCUSSION TITLE
Issue with 1D filter scales

### DISCUSSION THREAD

**Discussion by LisaCuneo** [kymatio/kymatio#1050] [Q&A]
I would like to process a sequence of images (a video) using scattering networks. Since I couldn't find a proper 3D (or rather, 2D+1D) example for video analysis, I started generating the filters as the tensor product of 2D and 1D filters. By following the examples on this GitHub page, I created the necessary structure to compute the 2D+1D scattering coefficients. In doing so, I encountered the following oddity that I cannot explain: given Q=(1,1), why do the 1D psi filters not have increasing scales j but instead repeat the filter at j=0 with varying values of x_i and sigma? By varying Q, the number of filters at j=0 changes, but since I cannot use a fraction for Q, I never manage to get one filter per scale as I would expect. The role of 1D scale j is relevant for the aforementioned application in 2D+1D, as it affects which paths are frequency-decreasing path. I need to understand the reason for having multiple 1D filters at the same scale to calculate the final number of scattering coefficients up to the second order (m=2), given the spatial scale J, the number of angles L, and the temporal scale S.

If you are interested, I can make a pull request with my code on your GitHub page.

**Comment by MuawizChaudhary** [comment]
@lostanlen can prolly elucidate on this issue. 

Paging @eickenberg also as he's investigated scattering for videos before.
